### PR TITLE
Add location routes and API

### LINKED ISSUE #5013: No Location Routes to the api available
Continuation from https://github.com/fossasia/open-event-server/pull/4982
**Describe the bug**

Frontend needs a route (an URL) to call the locations in order for them to be loaded. and displayed in the footer.

See frontend: https://github.com/fossasia/open-event-frontend/pull/1304

### PR DESCRIPTION
**Pull request by Kreijstal**
Fixes #5013 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Allows locations to be called from the API

#### Changes proposed in this pull request:

-Adds routes to event-locations
-And binds it with the existing models

### CODE REVIEW

File: populate_db.py [hunk 1]
```diff
@@ -120,6 +120,11 @@ def create_event_types():
         get_or_create(EventType, name=type_)
 
 
+def create_event_locations():
+    event_location = ['India', 'Singapore', 'Berlin', 'New York', 'Hong Kong']
+    for loc_ in event_location:
+        get_or_create(EventLocation, name=loc_)
+
```
Comment (bhaveshAn) [inline populate_db.py]: `create_event_locations` is added by you already :smile:

Comment (Kreijstal) [reply populate_db.py]: AHHHHHHHHHHH

Comment (Kreijstal) [reply populate_db.py]: @bhaveshAn Thank you, I don't know how that got there.

File: docs/api/api_blueprint.apib [hunk 2]
```diff
@@ -1215,6 +1215,91 @@ Get the details of the user.
...
+Event Locations for Events. 
+
+
+| Parameter | Description | Type | Required |
+|:----------|-------------|------|----------|
+| `name`  | Name of the event type | string | - |
+| `slug` | Slug of the event type | string | - |
+
+
+## Event Locations Collection [/v1/event-types{?page%5bsize%5d,page%5bnumber%5d,sort}]
```
Comment (poush) [inline docs/api/api_blueprint.apib]: is the url correct here?

File: app/api/event_locations.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    """
+    List event locations
+    """
+    decorators = (api.has_permission('is_admin', methods="POST"),)
+    schema = EventLocationSchema
+    data_layer = {'session': db.session,
+                  'model': EventLocation}
+
+
+class EventLocationDetail(ResourceDetail):
```
Comment (poush) [inline app/api/event_locations.py]: I don't see use of EventLocationDetail anywhere in your PR.

**Comment by Kreijstal** [comment]
@poush Removed it, Didn't know it was not needed.

File: tests/hook_main.py [hunk 4]
```diff
@@ -3053,6 +3053,16 @@ def file_upload_post(transaction):
...
 
+# ------------------------- Event Locationss -------------------------
+@hooks.before("Event Locations > Event Locations Collection > List All Event Locations")
+def event_location_get_list(transaction):
+    """
+    GET /events-location
+    :param transaction:
+    :return:
+    """
+
```
Comment (mayank8318) [inline tests/hook_main.py]: Don't you need to add an event here?

Comment (Kreijstal) [reply tests/hook_main.py]: event-locations is static.

**Comment by Kreijstal** [comment]
@poush there

File: docs/api/api_blueprint.apib [hunk 5]
```diff
@@ -1240,6 +1240,92 @@ Check if the email passed is available or not. True is returned if the email is
...
+
+
+## Event Locations Collection [/v1/event-locations{?page%5bsize%5d,page%5bnumber%5d,sort}]
++ Parameters
+    + page%5bsize%5d (optional, integer, `10`) - Maximum number of resources in a single paginated response.
+    + page%5bnumber%5d (optional, integer, `2`) - Page number to fetched for the paginated response.
+    + sort (optional, string, `name`) - Sort the resources according to the given attribute in ascending order. Append '-' to sort in descending order.
+
+### List All Event Locations [GET]
+Get a list of Event Types.
```
Comment (mayank8318) [inline docs/api/api_blueprint.apib]: Get a list of Event ~Types~ Locations

Comment (Kreijstal) [reply docs/api/api_blueprint.apib]: Good eye

**Pull request merged**